packages/foundry/contracts/test/oracles/ZenBullOracleTest.t.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
Please deploy on a fork, and use that to code a test harness that verifies that the ZenBullPrice obtained is within an order of magnitude of something. If ZenBull is expected to stay within 0.1 ETH and 10 ETH most of the time, that would be a good range to test against.